include/pm/pm.h

```diff
@@ -72,10 +72,11 @@ struct pm_notifier {
  *
  * @note This function can only run in thread context
  *
+ * @param cpu CPU index.
  * @param info Power state which should be used in the ongoing
  *	suspend operation.
  */
-void pm_power_state_force(struct pm_state_info info);
+void pm_power_state_force(int cpu, struct pm_state_info info);
```

Comment:
I observe that the [cpu] parameter is using the signed integer type. Would it be both better and more consistent to use an unsigned integer type? I ask this for the following reasons ...

1. A negative CPU ID does not make much immediate sense to me. (I may of course be missing a use case.)
2. The CPU id stored within "struct _cpu" is of type uint8_t. So keeping the unsigned quality helps avoid signed/unsigned mix-and-match.

Reply:
I followed some API that use `int` for this purpose, but I really think you are right about it.